yb-voyager/cmd/importSchema.go

```diff
@@ -110,9 +111,39 @@ func importSchema() {
 	}
 
 	log.Info("Schema import is complete.")
+	if isMViewsPresent {
+		utils.PrintAndLog("Materialised Views are present in the imported schema, use --refresh-mviews flag with --post-import-data to refresh mviews after importing data.\n\n")
+	}
+	if flagPostImportData && flagRefreshMViews  {
+		refreshMViewsPostImportData(conn)
+	}
 	callhome.PackAndSendPayload(exportDir)
 }
 
+func refreshMViewsPostImportData(conn *pgx.Conn) {
+	utils.PrintAndLog("\nRefreshing Materialised Views..\n\n")
+	var listOfMViewsImported []string
+	schemaDir := filepath.Join(exportDir, "schema")
+	importMViewFilePath := utils.GetObjectFilePath(schemaDir, "MVIEW")
+	if !utils.FileOrFolderExists(importMViewFilePath) {
+		return
+	}
+	mviewsSqlInfoArr := createSqlStrInfoArray(importMViewFilePath, "MVIEW")
+	for _, eachMviewSql := range mviewsSqlInfoArr {
+		if strings.Contains(eachMviewSql.stmt, "CREATE MATERIALIZED VIEW") {
```

Comment:
comparison with `eachMviewSql.stmt` should be case-insensitive. It might be possible that user has added/edited the stmt and it is like this `create materialized view`